### PR TITLE
Added `--host` support

### PR DESCRIPTION
**Pull request by jwhitcraft**
- For when you need to change the ip address.
- Added Tests
- Updated README

### CODE REVIEW

File: lib/guard/rack/runner.rb [hunk 1]
```diff
@@ -41,7 +41,8 @@ def build_rack_command
       command.push(
         options[:config],
         '--env', options[:environment].to_s,
-        '--port', options[:port].to_s
+        '--port', options[:port].to_s,
+        '--host', options[:host].to_s
```
Comment (dblock) [inline lib/guard/rack/runner.rb]: Nitpick: swap host and port, typically options host comes first.

**Comment by dblock** [comment]
This is great, thanks. Please update CHANGELOG as well, and squash your commits.

File: lib/guard/rack/runner.rb (hunk 1 above)
Comment (jwhitcraft) [inline lib/guard/rack/runner.rb]: This is taken care of: https://github.com/dblock/guard-rack/pull/17/files#diff-46b6856b2f57a5f4eaf3d1e192a5469cR12

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 2.1.0 (Next)
 ============
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Don't delete this, leave it underneath for the next person.

**Comment by dblock** [comment]
Thanks! Merging.

**Pull request merged**